modules/Microsoft.Network/frontDoors/readme.md

```diff
@@ -22,32 +22,32 @@ This module deploys Front Doors.
 ## Parameters
 
 **Required parameters**
-| Parameter Name | Type | Default Value | Description |
-| :-- | :-- | :-- | :-- |
-| `backendPools` | array | `[]` | Backend address pool of the frontdoor resource. |
-| `enabledState` | string | `'Enabled'` | State of the frontdoor resource. |
-| `friendlyName` | string | `''` | Friendly name of the frontdoor resource. |
-| `frontendEndpoints` | array | `[]` | Frontend endpoints of the frontdoor resource. |
-| `healthProbeSettings` | array | `[]` | Heath probe settings of the frontdoor resource. |
-| `loadBalancingSettings` | array | `[]` | Load balancing settings of the frontdoor resource. |
-| `name` | string |  | The name of the frontDoor. |
-| `routingRules` | array | `[]` | Routing rules settings of the frontdoor resource. |
+| Parameter Name | Type | Description |
+| :-- | :-- | :-- |
+| `name` | string | The name of the frontDoor. |
```

Comment:
A lot of optional parameters with an empty value. Not in this PR but we should add a min deployment test sooner than later to validate if they are really optional for a frontdoor.

Reply:
Updated to older API version